sway/main.c

```diff
@@ -25,10 +28,6 @@ int main(int argc, char **argv) {
 	}
 	setenv("DISPLAY", ":1", 1);
 
-	if (!load_config()) {
-		sway_abort("Unable to load config");
-	}
-
```

Comment:
We need to run exec after wlc is initialized so that x programs will have a server to use.